src/components/templates/Playa/Playa.tsx

```diff
@@ -41,7 +41,7 @@ import { WithId } from "utils/id";
 // import { updateLocationData } from "utils/userLocation";
```

Comment:
[note] Playa is legacy code to be removed so we don't really care if it breaks here.